### DISCUSSION TITLE
提示： 发生身份验证错误。怎么解决？

### DISCUSSION THREAD

**Discussion by yup1212** [natfrp/wiki#199] [使用问题] [answered]
完整的提示：
发生身份验证错误。
给函数提供的标志无效
远程计算机：XXXXXXXXXXXXX.natfrp.cloud

这个问题什么原因？怎么解决？

**Comment by Archeb** [comment]
请参考关于CredSSP导致的RDP连接失败：https://cloud.tencent.com/document/product/213/30813

**Comment by Archeb** [comment, marked as answer]
如果您使用的是枣庄节点，请参考我们的通知 

> **枣庄多线节点屏蔽 TLS 通知**
> 我们已确认所有枣庄多线节点均屏蔽了高位端口的 TLS 流量。由于这是机房直接进行的屏蔽，我们对此无能为力，且短时间内屏蔽不会解除。
> 
> 受影响的常见业务包括 RDP、高位端口的 HTTPS 流量、访问认证功能等，建议需要使用这些业务的用户迁移到其他节点。
